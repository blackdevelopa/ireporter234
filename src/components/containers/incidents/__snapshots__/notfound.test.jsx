import React from 'react';
import { shallow } from 'enzyme';
import Notfound from '../NotFound';

describe('<NotFound >', () => {
  const wrapper = shallow(<Notfound />);
  it('should render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
