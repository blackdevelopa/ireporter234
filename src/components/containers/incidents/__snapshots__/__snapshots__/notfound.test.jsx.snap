// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound > should render without crashing 1`] = `
<div>
  <Navbar
    access="true"
    firstbtn="Redflag"
    secondbtn="Intervention"
  />
  <h1
    style={
      Object {
        "fontSize": "40px",
        "textAlign": "center",
      }
    }
  >
    Sorry. We think a bear ate the page you are looking for.
  </h1>
</div>
`;
